ts/Series/FlowMap/FlowMapSeries.ts

```diff
@@ -77,20 +86,13 @@ class FlowMapSeries extends SankeySeries {
         },
 
         /**
-         * DESCRIPTION
-         *
          * @declare Highcharts.SeriesFlowMapSeriesDataLabelsOptionsObject
          *
+         * @extends series.mappoint.keys
+         * @default ['from', 'to', 'weight']
          * @private
          */
-        keys: [
```

Comment:
Why has this been changed here? Shouldn't the default be: 'from', 'to', 'curveFactor', 'weight', 'growTowards', 'markerEnd'

Reply:
@KacperMadej suggested, that `curveFactor`, `growTowards`, `markerEnd` should be set as default from the series object, so it shouldn't be in `keys` property by default.